e2e/node/wac.test.ts

```diff
@@ -65,10 +67,12 @@ describe("Authenticated end-to-end WAC", () => {
   let options: { fetch: typeof global.fetch };
   let session: Session;
   let sessionResource: string;
+  let pod: string;
 
   beforeEach(async () => {
     session = await getAuthenticatedSession(env);
-    sessionResource = `${env.pod}${sessionResourcePrefix}${session.info.sessionId}`;
+    pod = await getPodRoot(session);
+    sessionResource = `${pod}${sessionResourcePrefix}${session.info.sessionId}`;
```

Comment:
@NSeydoux @chelseapinka something we'll want to be very careful of in the future is making sure we construct IRIs properly with `new URL` it's not clear that `pod` has a trailing slash

Reply:
Ah good point I missed that